archetype-java-builders/src/main/resources/archetype-resources/src/main/java/ExampleJLoadScenarioProvider.java

```diff
@@ -34,33 +33,37 @@
 import com.griddynamics.jagger.user.test.configurations.termination.JTerminationCriteriaIterations;
 import com.griddynamics.jagger.user.test.configurations.termination.auxiliary.IterationsNumber;
 import com.griddynamics.jagger.user.test.configurations.termination.auxiliary.MaxDurationInSeconds;
-import com.griddynamics.jagger.util.JaggerPropertiesProvider;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import ${package}.util.JaggerPropertiesProvider;
 
 /**
- * By extending {@link JaggerPropertiesProvider} you get access to all Jagger properties, which you can use
- * for configuration of JLoadScenario.<p>
- * Benefit of this approach is that you can change JLoadScenario configuration by changing properties file and no recompilation is needed.
+ * By extending {@link JaggerPropertiesProvider} you get access to all Jagger properties and test properties, which are needed for load scenario
+ * configuration, but must not get to the Jagger environment. You can use them for configuration of JLoadScenario.<p>
```

Comment:
please remove ", which are needed for load scenario
 - * for configuration of JLoadScenario.<p>		 + * configuration, but must not get to the Jagger environment"